src/Service.Tests/SqlTests/RestApiTests/Put/MsSqlPutApiTests.cs

```diff
@@ -233,6 +251,81 @@ public async Task PutOneInViewBadRequest()
             await base.PutOneInViewBadRequest(expectedErrorMessage);
         }
 
+        /// <summary>
+        /// Test to validate that even when an update DML trigger is enabled on a table, we still return the
+        /// latest data as it is present after the trigger gets executed. Whenever an update DML trigger is enabled,
+        /// we use a subsequent SELECT query to get the data instead of using OUTPUT clause.
+        /// </summary>
```

Comment:
It would be helpful to identify how the validation is done: ensured that even though salary was provided as -9, the response contains 0 due to the effect of the update trigger.

Reply:
Added calrification to comment.